engine/access/ingestion/engine.go

```diff
@@ -528,71 +530,6 @@ func (e *Engine) trackExecutedMetricForBlock(block *flow.Block, ti time.Time) {
 	}
 }
 
-func (e *Engine) trackExecutedMetricForCollection(light *flow.LightCollection) {
```

Comment:
can you also update this engine to use the `CollectionExecutedMetric` and move any other functions that are involved. `trackExecutedMetricForBlock` for example